components/badge/badge.css

```diff
@@ -0,0 +1,40 @@
+.badge__demo {
+  font-size: 2rem;
+  position: relative;
+  padding: 1rem 2rem;
+  width: fit-content;
+  font-family: var(--secondary-font-family);
+}
+
+.badge__demo-btn {
+  font-family: var(--secondary-font-family);
+  font-size: 2rem;
+  background-color: var(--primary-color);
+  border-color: transparent;
+  padding: 5px 10px;
+  border-radius: 5px;
+  color: var(--white-color);
+  cursor: pointer;
+}
+
+.badge__demo-txt {
+  margin-right: 0.5rem;
+}
```

Comment:
Don't use rem and px together. 
Stick to rem and define the root font size to 62.5% which makes 1rem = 10px.

Reply:
hey i have made the suggested changes can you look it up